### DISCUSSION TITLE
Clarification on Component health check

### DISCUSSION THREAD

**Discussion by Tonicku** [reportportal/reportportal#1649] [Ideas]
Hi,

I was playing with the Widget component Health, especially on Level.

I was expecting to have tree sorted by level.
if level 1, level2, level3 provided
then
result
      level 1 > level2 > level3

level o1 
    Level2 o1i1
        Level 3
            Level 3 o1i1j1 result o1i1j1
            ...
            Level 3 o1i1jn result o1i1jn
    .......
    Level2 o1in
        Level 3
            Level 3 o1ink1 result i1ink1
            ...
            Level 3 o1inkn result o1inkn
  ....
level on
      Level2 onp1
            Level 3 onp1q1   result onp1q1
            ...
            Level 3 onpn1n   result onpn1n
      Level2 onpn
            Level 3 onpnr1     result onpnr1
             ....
            Level 3 onpnrn     result onpnrn

but what i see is

  level 1
       level : contact of all level2
             level2 don't see anything
                      then results

![image](https://user-images.githubusercontent.com/69209540/151720401-ab1f1164-343d-4e9d-9e44-09fdaa793930.png)

are you planning to enhance this Component Health ?

Regards
Anthony

**Comment by Tonicku** [comment]
i found out that we need to click on the Level1 value to see result level 1 ->group by level 2 result.

are you planning to show the result, in not so many level, in one view ?

**Reply by DzmitryHumianiuk** [reply]
@Tonicku, no, we are not planning to display all the levels in one view. 
Doing so would contradict the concept of the `Component Health Check` widget. 
You can read about its intended design here: [Component Health Check Documentation](https://reportportal.io/docs/dashboards-and-widgets/ComponentHealthCheck).

It seems you might be referring to the tree view of this structure, which aligns more with the launch-level view. 
Please note that this will be added with the reworked UI.

**Comment by DzmitryHumianiuk** [comment]
@
